### PR TITLE
chore: Breaking changes page for 3.7

### PR DESCRIPTION
**Pull request by lena-larionova**
### Description

Breaking changes in 3.7.

https://konghq.atlassian.net/browse/DOCU-3808

### Testing instructions

Preview link: 

### Checklist 

- [x] Review label added 
- [x] [Conditional version tags](https://docs.konghq.com/contributing/conditional-rendering/#conditionally-render-content-by-version) added, if applicable.

### CODE REVIEW

File: app/_src/gateway/breaking-changes/37x.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+You may need to adopt different [upgrade paths](/gateway/{{page.release}}/upgrade/#guaranteed-upgrade-paths) depending on your 
+deployment methods, set of features in use, or custom plugins, for example.
+
+Review the [changelog](/gateway/changelog/#3700) for all the changes in this release.
+
+## Breaking changes and deprecations
+
+### Configuration
+
+The Granular Tracing feature has been removed from {{site.base_gateway}}.
```
Comment (Guaris) [inline app/_src/gateway/breaking-changes/37x.md:20]:
```suggestion
The Granular Tracing feature has been deprecated and removed from {{site.base_gateway}}.
```

Comment (Guaris) [reply app/_src/gateway/breaking-changes/37x.md:20]: I'm not sure if it was actually deprecated but listing it as 'removed' left me wanting more info

File: app/_src/gateway/breaking-changes/37x.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+The following tracing-related `kong.conf` parameters are no longer available. 
+Remove them from your configuration when upgrading to 3.7:
```
Comment (Guaris) [inline app/_src/gateway/breaking-changes/37x.md:23]:
```suggestion
As part of your upgrade to 3.7, please remove the following tracing-related parameters from your `kong.conf` file.
```

File: app/_src/gateway/breaking-changes/37x.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+* `tracing`
+* `tracing_write_strategy`
+* `tracing_write_endpoint`
+* `tracing_time_threshold`
+* `tracing_types`
+* `tracing_debug_header`
+* `generate_trace_details`
+
+You should now use [OpenTelemetry Instrumentation](/gateway/{{page.release}}/production/tracing/) instead.
```
Comment (Guaris) [inline app/_src/gateway/breaking-changes/37x.md:33]:
```suggestion
We recommend transitioning to [OpenTelemetry Instrumentation](/gateway/{{page.release}}/production/tracing/) instead.
```

File: app/_src/gateway/breaking-changes/37x.md (hunk 1 above)
Comment (lena-larionova) [reply app/_src/gateway/breaking-changes/37x.md:20]: It's both. It was deprecated in 3.0: https://docs.konghq.com/gateway/changelog/#breaking-changes-and-deprecations-5:~:text=Support%20for%20the%20nginx%2Dopentracing%20module%20is%20deprecated%20as%20of%203.0%20and%20will%20be%20removed%20from%20Kong%20in%204.0%20(see%20the%20Known%20Limitations%20section%20for%20additional%20information).

It's now fully removed (was supposed to be in 4.0, but seems like they did it earlier).

**Review by cloudjumpercat — APPROVED** [review]
LGTM!

**Pull request merged**